libs/langchain/langchain/callbacks/tracers/evaluation.py

```diff
@@ -81,7 +82,13 @@ def __init__(
         )
         self.client = client or langchain_tracer.get_client()
         self.evaluators = evaluators
-        self.executor = _get_executor()
+        if max_concurrency is None:
+            self.executor = _get_executor()
+        elif max_concurrency > 0:
+            self.executor = ThreadPoolExecutor(max_workers=max_concurrency)
+            weakref.finalize(self, self.executor.shutdown, False)
```

Comment:
is there a way to pass the `False` as a named argument?

Reply:
Could use a lambda ya

Reply:
Actually should have it be true